src/videojs-http-streaming.js

```diff
@@ -508,12 +509,9 @@ class HlsHandler extends Component {
     this.options_.tech = this.tech_;
     this.options_.externHls = Hls;
     this.options_.sourceType = simpleTypeFromSourceType(type);
-    // Whenever we seek internally, we should update both the tech and call our own
-    // setCurrentTime function. This is needed because "seeking" events aren't always
-    // reliable. External seeks (via the player object) are handled via middleware.
+    // Whenever we seek internally, we should update the tech
     this.options_.seekTo = (time) => {
```

Comment:
Is `seekTo` still used anywhere?

Reply:
Yes, it's currently used inside of seekToProgramTime and related functions.

Reply:
Should we then just leave all of the other uses of `seekTo`?

Reply:
We could, we specifically need `seekTo` for seekToProgramTime because STPT doesn't have access to anything else like the tech or what not.

Reply:
Might be worth keeping it consistent then. Also hides the use of tech a little more.